### DISCUSSION TITLE
How to navigate horizontally (using the "tabs")?

### DISCUSSION THREAD

**Discussion by esbenboye** [jesseduffield/lazygit#1619] [Q&A] [answered]
When starting LG, the 2nd window is pointing to Files, the 3rd is pointing to 'Local branches' and the 3rd is pointing to Commits. 
But how do I switch between the "tabs"? I have tried left/right and tabbing and I cant find anything in the docs.

I'm on MacOS if that matters.

**Comment by hynek** [comment, marked as answer]
`[` and `]`

**Reply by esbenboye** [reply]
Brilliant - Thank you!
